### DISCUSSION TITLE
Is the "Language" input field in a book's Details for language codes only?

### DISCUSSION THREAD

**Discussion by GreatestFool** [advplyr/audiobookshelf#2606] [Q&A] [answered]
I couldn't find an issue or a discussion regarding this, but I may have missed it. Couldn't find anything clear enough in the docs either.

In a book's Details, there's a field for "Language," is this for language codes only, e.g. en-US, or is it for natural language as a form of keyword hint, e.g. English?

I mean this:
![image](https://github.com/advplyr/audiobookshelf/assets/3003484/8b90d1c4-ae06-4e24-84f7-3c5c25eceba6)

Or this:
![image](https://github.com/advplyr/audiobookshelf/assets/3003484/39488cc8-df27-4ed9-9443-aaf8de962146)

**Comment by nichwall** [comment, marked as answer]
Either way. It's a string input so you can put whatever you want there.

For most of my books I'm using the full language name "English". But, I haven't taken the time to standardize everything to English since everything on my server is English.

The language code might be better if you're also using other software or embedding all of the metadata back in and the other software expects language codes.

**Reply by nichwall** [reply]
You can search by language in the library view, so that's the most important part from an ABS perspective. You'll probably have some random codes in there, like the "unknown: xxx"

![Screenshot_20240212-070026.png](https://github.com/advplyr/audiobookshelf/assets/5686638/9b81620c-cb62-48cb-8d46-f7f5c5d6bc7e)
